### ISSUE TITLE
Apps don't show up without connecting an AWS account

### ISSUE THREAD

**Issue by sterling000** [serverless/desktop#20]
Hello, 

I have several apps across multiple organizations that i've deployed with app.serverless.com.

When I login to the new desktop app, I see my organization list in the bottom right, but none of my apps are listed.

My apps are deployed using IAM User accounts created by my client's root accounts, so I would prefer to not have to login with new credentials than the ones i've already used.

I do my deploys from command line so far with access keys, I would prefer to enter them into the desktop app but have the lsit of my apps loaded on first launch and when i switch organizations.

**Comment by stevewillard** [comment]
Hi there,

What region are your apps deployed to? Currently only these regions are supported:
```
us-east-1'
us-east-2'
us-west-1'
us-west-2'
ca-central-1
eu-central-1
eu-west-1
eu-west-2
eu-west-3
eu-north-1
```

At the moment, Desktop requires that you create a new IAM role within the account where your Serverless apps are deployed. Thanks for the feedback on access keys - we had this functionality at one point, but decided to remove it to simplify the permissions and setup process. We may re-introduce it.

**Comment by sterling000** [comment]
Right, I didn't see anything about he region limit when I installed and logged in so thought it was weird, most of my current apps are in ap-southeast-1, but I do have one website component in us-east-1 that doesn't show up either.
